README.md

```diff
@@ -125,48 +124,51 @@ $givenNames         = $profile->getGivenNames()->getValue();
 
 $fullName           = $profile->getFullName()->getValue();
 
-$dateOfBirth        = $profile->getDateOfBirth()->getValue();
+$dateOfBirth        = $profile->getDateOfBirth()->getValue(); // DateTime Object
 
 $gender             = $profile->getGender()->getValue();
 
 $nationality        = $profile->getNationality()->getValue();
 
 $phoneNumber        = $profile->getPhoneNumber()->getValue();
 
-$selfie             = $profile->getSelfie()->getValue();
+$selfie             = $profile->getSelfie()->getValue(); // Yoti\Entity\Image Object
 
 $emailAddress       = $profile->getEmailAddress()->getValue();
 
 $postalAddress      = $profile->getPostalAddress()->getValue();
 
-$ageVerified        = $profile->getAgeCondition()->getValue();
+$ageVerifications   = $profile->getAgeVerifications(); // array of AgeVerification object, e.g ['age_over:18' => new AgeVerification(),]
+
+$ageOverXX          = $profile->findAgeOverVerification($xx); // AgeVerification Object (please see API Coverage for details), Where $xx is the age value, e.g 50
+
+$ageUnderXX         = $profile->findAgeUnderVerification($xx); // AgeVerification Object (please see API Coverage for details), Where $xx is the age value, e.g 18
 
-$verifiedAge        = $profile->getVerifiedAge()->getValue();
 ```
 
 ## Handling Users
 
-When you retrieve the user profile, you receive a user ID generated by Yoti exclusively for your application.
+When you retrieve the user profile, you receive a rememberMe ID generated by Yoti exclusively for your application.
 This means that if the same individual logs into another app, Yoti will assign her/him a different ID.
 You can use this ID to verify whether (for your application) the retrieved profile identifies a new or an existing user.
 Here is an example of how this works:
 
 ```php
 <?php
-$activityDetails = $client->getActivityDetails($token);
-$profile = $activityDetails->getProfile();
-
-if ($client->getOutcome() == \Yoti\YotiClient::OUTCOME_SUCCESS) {
-    $user = yourUserSearchFunction($activityDetails->getUserId());
+try {
+    $activityDetails = $client->getActivityDetails($token);
```

Comment:
Could we change this to `oneTimeUseToken` as per https://lampkicking.atlassian.net/browse/SDK-209? (I know it still has to be `token` in the get request). Also we mention `token` other times in the README, please can you change those too?

Reply:
Not sure if using $oneTimeUseToken will be clearer as than the comments I have added [here](https://github.com/getyoti/yoti-php-sdk/tree/SDK-649#profile-retrieval)